### PR TITLE
[Replace Non-Blocking Usage] LPS-202096 Replace blocking ServiceProxyFactory with Snapshot in class  AutoBatchPreparedStatementUtil

### PR DESCRIPTION
**Pull request by JanisZhang**
https://liferay.atlassian.net/browse/LPS-202096

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:test:upgrade

**Comment by JanisZhang** [comment]
ci:test:startup

**Comment by JanisZhang** [comment]
ci:test:relevant

File: portal-kernel/src/com/liferay/portal/kernel/dao/jdbc/AutoBatchPreparedStatementUtil.java [hunk 1]
```diff
@@ -255,11 +258,11 @@ private ConcurrentNoBatchInvocationHandler(
 			super(connection, sql);
 		}
 
+		private static final PortalExecutorManager _portalExecutorManager =
```
Comment (dantewang) [inline portal-kernel/src/com/liferay/portal/kernel/dao/jdbc/AutoBatchPreparedStatementUtil.java:261]: @JanisZhang Please keep it consistent with before -- no `static`.

**Review by dantewang — CHANGES_REQUESTED** [review]
See my comment.

**Comment by JanisZhang** [comment]
ci:stop

File: portal-kernel/src/com/liferay/portal/kernel/dao/jdbc/AutoBatchPreparedStatementUtil.java (hunk 1 above)
Comment (JanisZhang) [reply portal-kernel/src/com/liferay/portal/kernel/dao/jdbc/AutoBatchPreparedStatementUtil.java]: My bad. Force-pushed. Thanks!

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:upgrade

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by JanisZhang** [comment]
ci:test:startup

**Review by dantewang — APPROVED** [review]

**Comment by zhangyan0701** [comment]
No regressions found.

**Pull request closed without merging**